packages/web-scripts/config/commitlint.config.js

```diff
@@ -6,5 +6,23 @@ module.exports = {
       'never',
       ['sentence-case', 'start-case', 'pascal-case', 'upper-case'],
     ],
+    'type-enum': [
+      1,
+      'always',
+      [
+        'feat',
+        'fix',
+        'improvement',
```

Comment:
improvement and perf are in the new commitizen prompter (which i think looks quite nice!), but are oddly not in the commitlint enum by default.